test/functional/testSettings.js

```diff
@@ -0,0 +1,739 @@
+// https://github.com/elastic/kibana-qa/blob/master/settings.md
```

Comment:
Yes.  That settings.md set of tests goes way beyond what the testSettings test currently do.  testSettings covers most of the first 2 sections of that md ('Adding an index pattern', and 'Index pattern field list').  I could update that .md to say that once this is merged and working.